draft-ietf-babel-information-model.md

```diff
@@ -279,40 +376,67 @@ babel-supported-link-types:
 
 babel-self-seqno:
 : The current sequence number included in route updates for routes
-  originated by this node.
+  originated by this node. This is a 16-bit unsigned integer.
 
 babel-metric-comp-algorithms:
 : List of supported cost computation algorithms. Possible
   values include "k-out-of-j", and "ETX".
 
 babel-security-supported:
-: List of supported security mechanisms. As Babel security mechanisms
-  are defined, they will need to indicate what enumeration value is to
-  be used to represent them in this parameter.
+: List of supported security mechanisms. Possible values include
+  "HMAC" and "DTLS".
+
+babel-hmac-enable:
+: Indicates whether the HMAC security mechanism is enabled
+  (true) or disabled (false).
+  An implementation MAY choose
+  to expose this parameter as read-only ("ro").

```

Comment:
Some implementations of the info model may be for reporting purposes, only. For example, a web page. The expectation there is that to configure the function, you need to use some different interface (which may be SSH, netconf, TR-069). Data model implementations like YANG or TR-181 
will have to define these as "rw" because they are intended to be used for reporting and configuration. There was an old comment that people didn't want "report only" implementations of the info model precluded. Just because a "report only" implementation exists doesn't mean there is no (other) way to configure functionality.

Reply:
Since YANG and BBF TR-181 models aren't impacted by these "MAY choose to expose as ro" statements, maybe we can agree to hold off on resolving this issue for now? Somehow it needs to be clear that report-only data models can exist (as long as there is some way to do configuration, somehow). We can figure out the right language later without impacting the actual model (or YANG or TR-181).

Reply:
Ok. An attribute can always be made ro, it the reverse that is more difficult to do, at least in YANG. By that, I mean, when an attribute is marked ro, it means that under no circumstances should it be writeable. Whereas, a rw attribute can be restricted to ro by an implementation. And that is why my concern. But if there is a specific requirement for this, in YANG I will choose to drop the last line of the description, and keep the attribute as rw. Just letting you know.